cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml

```diff
@@ -16,3 +16,17 @@ suites:
       resource: chrony:enable
       dependencies:
       - resource:chrony:setup
+  - name: ssh_target_checker
+    run_list:
+      - recipe[aws-parallelcluster-platform::openssh]
+    verifier:
+      controls:
+        - /ssh_target_checker_script_created/
```

Comment:
Nitpick: you don't need to use the regexp here,  as you need to run exactly that specific control. You can therefore remove the slashes around the control name.

Reply:
I have added one control for Docker and another for EC2 which is why I have used a Regex.